packages/terra-core-docs/src/terra-dev-site/test/pills/pill-list/CollapsiblePillListWithDifferentPillType.test.jsx

```diff
@@ -0,0 +1,61 @@
+import React, { useState } from 'react';
+import classNames from 'classnames/bind';
+import { Button } from '@cerner/terra-docs';
+import PillList, { Pill } from 'terra-pills';
+import styles from './PillListTestCommon.module.scss';
+
+const cx = classNames.bind(styles);
+
+const CollapsiblePillListWithDifferentPillType = () => {
+  const [isCollapsed, setIsCollapsed] = useState(true);
+
+  const handleRollUpPillOnClick = () => {
+    setIsCollapsed(false);
+  };
+
+  const handleButtonOnClick = () => {
+    setIsCollapsed(!isCollapsed);
+  };
+
+  const handleOnRemove = () => {
+    // Callback just to enable remove button
+  };
+
+  const handleOnSelect = () => {
+    // Callback just to make pill selectable
+  };
+
+  return (
+    <>
+      <PillList
+        ariaLabel="Pill Container"
+        isCollapsed={isCollapsed}
+        rollUpPillOnTrigger={handleRollUpPillOnClick}
+        className={cx(['show-border', 'width-200'])}
+      >
+        <Pill
```

Comment:
Should we even allow different pills types in the same pilllist? It seems to make for a very wonky keyboard navigation pattern. I wonder if we should elevate the types to the pilllist and either have different pilllists or elevate teh on select and on remove to the pill list and have it control the type of pills.